cluster/gce/util.sh

```diff
@@ -771,39 +771,78 @@ function create-nodes() {
 # - NODE_INSTANCE_PREFIX
 # - PROJECT
 # - ZONE
+# - AUTOSCALER_MAX_NODES
+# - AUTOSCALER_MIN_NODES
+function create-cluster-autoscaler-mig-config() {
+
+  # Each MIG must have at least one node.
+  if $((${AUTOSCALER_MIN_NODES} < ${NUM_MIGS})); then
```

Comment:
Ok, now I understand. Please explain why this actually ensures that each MIG will have at least 1 node.